code/modules/cargo/department_order.dm

```diff
@@ -191,6 +191,7 @@ GLOBAL_LIST_INIT(department_order_cooldowns, list(
 	name = "security order console"
 	circuit = /obj/item/circuitboard/computer/security_orders
 	department_delivery_areas = list(
+		/area/station/security/office,
 		/area/station/security/brig,
 		/area/station/security/brig/upper,
```

Comment:
better to alphabetize IMO, I don't think this list is weighted at all.

Reply:
it currently works as a priority list, it only doesn't pick the top option if it doesn't exist.

Reply:
ah, didn't know that. thanks